atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -510,7 +523,10 @@ private void finishConsumingPreviousRow(PeekingIterator<Map.Entry<Cell, Value>>
                                                                   Map<Cell, Value> rawResults,
                                                                   ImmutableMap.Builder<Cell, byte[]> result) {
         getWithPostFiltering(tableRef, rawResults, result, Value.GET_VALUE);
+        Map<Cell, byte[]> withDeletedValues = result.build();
```

Comment:
code reuse between here and https://github.com/palantir/atlasdb/pull/3977/files#diff-bf062bd4c13b8c0022080ca8ef49c4d2R586